libs/block-scripts/shared/block-assets-plugin.js

```diff
@@ -19,7 +19,7 @@ export class BlockAssetsPlugin {
         "assets-manifest.json",
       );
 
-      const defaultManifest = { "main.js": "main.js" };
+      const defaultManifest = { "main.cjs": "main.cjs" };
```

Comment:
I had to change the file-endings to `.cjs` because the `package.json` was polluting the scripts and saying they were ES Module scripts. 

This wasn't a problem when publishing the block, but it did end up being a problem when trying to import `example-graph.js`

Reply:
So.. I removed this and everything worked, I am not sure why as I don't believe I've changed anything about `"type": "module"` or any of that. 

Regardless, avoiding the file-endings seems nicer so I've committed it. cc: @CiaranMn